### DISCUSSION TITLE
I am not able to clone repository in my local machine

### DISCUSSION THREAD

**Discussion by Akki17** [DevSkillsHQ/discusssions#51] [Help] [answered]
I am not able to clone repository in my local machine

**Comment by Akki17** [comment]
how to solve this'

**Comment by fenske** [comment, marked as answer]
Hi @Akki17! You can either check out [this article](https://www.notion.so/devskills/I-can-t-clone-my-assignment-repository-749b58bbd7d9449a91aa3a091bcf6590) or use GitHub Codespaces as described in the README.
